server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java

```diff
@@ -94,7 +94,7 @@ public final User generateAdministrator(Organizations.Organization organization,
   @SafeVarargs
   public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
+    session.wsClient().organizationsOld().addMember(DEFAULT_ORGANIZATION_KEY, user.getLogin());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "organizationsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)